### PR TITLE
Eri9600/query map image sublayer

### PR DESCRIPTION
**Pull request by eri9600**
Query Map Image Sublayer sample as described here: https://devtopia.esri.com/runtime/common-samples/issues/561. I am using ArcGIS 100.3.0-2070.

### CODE REVIEW

**Comment by eri9600** [comment]
Hi @TADraeseke, can you review when you have time ? 
Thanks!

File: java/query-map-image-sublayer/src/main/java/com/esri/arcgisruntime/sample/querymapimagesublayer/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+        // query the sublayers when the button is clicked
+        mQueryButton.setOnClickListener(v -> {
+
+          // clear previous results
+          graphicsOverlay.getGraphics().clear();
+
+          // create query parameters filtering based on population and the map view's current viewpoint
+          QueryParameters populationQuery = new QueryParameters();
+          populationQuery.setWhereClause("POP2000 > " + mQueryInputBox.getText().toString());
```
Comment (TADraeseke) [inline java/query-map-image-sublayer/src/main/java/com/esri/arcgisruntime/sample/querymapimagesublayer/MainActivity.java]: Don't need .toString() here

File: java/settings.gradle [hunk 2]
```diff
@@ -62,6 +62,7 @@ include ':add-graphics-renderer',
         ':openstreetmap-layer',
         ':picture-marker-symbols',
         ':project',
+        ':query-map-image-sublayer',
```
Comment (TADraeseke) [inline java/settings.gradle]: put in alphabetical order (it may have been originally, but new samples have reordered things)

Comment (eri9600) [reply java/settings.gradle]: what do i put in alphabetical order? the recently added project or the whole file? To me the project looks in alphabetical order, maybe I am not seeing what you are. Thanks!

**Comment by eri9600** [comment]
@TADraeseke Fixed the suggestions you made. Let me know if i am missing something else.

File: java/settings.gradle (hunk 2 above)
Comment (TADraeseke) [reply java/settings.gradle]: just put query map image sublayer below portal user info :)

**Comment by eri9600** [comment]
@TADraeseke Alphabetized the file. Let me know if I missed anything else.

**Comment by TADraeseke** [comment]
Nice work @eri9600 - Thanks!

**Comment by eri9600** [comment]
@TADraeseke thanks for reviewing. Merging now.

**Pull request merged**